src/conduits/java/com/enderio/conduits/common/network/ConduitSavedData.java

```diff
@@ -25,13 +26,16 @@
 import net.minecraftforge.eventbus.api.SubscribeEvent;
 import net.minecraftforge.fml.common.Mod;
 
+import javax.annotation.Nullable;
 import java.util.*;
 
 @Mod.EventBusSubscriber
 public class ConduitSavedData extends SavedData {
 
     ListMultimap<IConduitType, Graph<Mergeable.Dummy>> networks = ArrayListMultimap.create();
 
+    public Map<IConduitType, Map<ChunkPos, Map<BlockPos, NodeIdentifier>>> deserializedNodes = new HashMap<>();
```

Comment:
make private and final, add comment explaining that this is for finding NodeIdentifier references for when chunk load